src/providers/BlockchainDotCom.test.ts

```diff
@@ -0,0 +1,21 @@
+import { getBTCBalance, getBTCPrice } from "./BlockchainDotCom"
```

Comment:
@martinvol 

Data gathering starts in this file down

Reply:
Do you have this functions mocked somewhere for tests? I can't see it

Reply:
ah yes using` msw` package to mock the actual responses here src/mocks/handlers.ts